### PR TITLE
getMediaElement private API

### PR DESCRIPTION
**Pull request by karimJWP**
### This PR will...
expose a getter to obtain the videoTag
### Why is this Pull Request needed?
to unblock Ads Team @pajong @jw-ads-team with their background loading work
### Are there any points in the code the reviewer needs to double check?
n/a
### Are there any Pull Requests open in other repos which need to be merged with this?
n/a
#### Addresses Issue(s):
JW8-857

### CODE REVIEW

File: src/js/api/api.js [hunk 1]
```diff
@@ -967,6 +967,14 @@ Object.assign(Api.prototype, /** @lends Api.prototype */ {
     },
 
     /**
+     * Extracts the video tag in the foreground.
+     * @returns {string} videoTag - the video tag in the foreground.
+     */
+    getVideoTag() {
+        return this.getContainer().getElementsByClassName('jw-media')[0].getElementsByTagName('video')[0];
```
Comment (johnBartos) [inline src/js/api/api.js]: What happens if there's no container, or `jw-media` containers, or a `video` within the container?

Comment (egreaves) [reply src/js/api/api.js]: The following works: 
```
const container = this.getContainer();
return container && container.querySelector('video'); // returns the 1st video element or null if there's none
```

Comment (karimJWP) [reply src/js/api/api.js]: addressed

**Comment by karimJWP** [comment]
test this please

File: src/js/api/api.js [hunk 2]
```diff
@@ -967,6 +967,15 @@ Object.assign(Api.prototype, /** @lends Api.prototype */ {
     },
 
     /**
+     * Extracts the video tag in the foreground.
+     * @returns {string} videoTag - the video tag in the foreground.
```
Comment (johnBartos) [inline src/js/api/api.js]: It doesn't return a string

File: src/js/api/api.js [hunk 3]
```diff
@@ -967,6 +967,15 @@ Object.assign(Api.prototype, /** @lends Api.prototype */ {
     },
 
     /**
+     * Extracts the video tag in the foreground.
+     * @returns {object|null} videoTag - the HTML <video> element in the foreground.
```
Comment (johnBartos) [inline src/js/api/api.js]: It's an `Element`, not object

https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector

File: src/js/api/api.js [hunk 4]
```diff
@@ -967,6 +967,15 @@ Object.assign(Api.prototype, /** @lends Api.prototype */ {
     },
 
     /**
+     * Extracts the video tag in the foreground.
+     * @returns {Element|null} videoTag - the HTML <video> element in the foreground.
```
Comment (johnBartos) [inline src/js/api/api.js]: It can also return undefined

**Pull request merged**